baseplate/experiments/__init__.py

```diff
@@ -172,54 +194,65 @@ def variant(self, name, user=None, bucketing_event_override=None,
         if do_log:
             assert bucketing_id
             self._log_bucketing_event(experiment, variant, user,
-                                      extra_event_fields)
+                                      inputs)
             self._already_bucketed.add(bucketing_id)
 
         return variant
 
-    def _log_bucketing_event(self, experiment, variant, user,
-                             extra_event_fields):
-        if not self._event_queue:
-            return
-
-        extra_event_fields = extra_event_fields or {}
+    def _log_bucketing_event(self, experiment, variant, user, extra_event_fields):
+
+        """ Logs a v2 bucketing event.
+
+            :param dict experiment: An experiment config dict.
+            :param string variant: The variant bucketed into
+            :param baseplate.core.User user: The user being bucketed. If this
+                is not provided, it is expected that the extra_event_fields
+                dict will contain the following information:
+                - user_id - fullname
+                - logged_in - boolean
+                - cookie_created_timestamp - epoch ms
+                - created_timestamp - epoch ms
+            :param dict extra_event_fields: addtional fields to be added. For
+                now, this expects and will process only the set of
+                user-related fields specified in the user parameter.
+        """
 
-        event = Event("bucketing_events", "bucket")
+        if not self._event_queue or not self._event_constructor:
+            logger.warning("missing event queue or bucketing constructor")
```

Comment:
`warning` is a pretty high criticality of log message. is this something we expect to be a standard case e.g. in dev (and so should be lower level)? or should it be even stronger and raise an exception?